From ea3246bc9a7461afadf4a48ca4c60068d08cbc40 Mon Sep 17 00:00:00 2001
From: Vincent Cao <vincent.t.cao@intel.com>
Date: Thu, 21 Nov 2019 16:37:55 -0800
Subject: [PATCH 011/109] tee: Export the tee core cancel func

Export of the cancel func to be useful for modules.

Signed-off-by: Vincent Cao <vincent.t.cao@intel.com>
---
 drivers/tee/tee_core.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/tee/tee_core.c b/drivers/tee/tee_core.c
index 85102d12d716..af7574a264c3 100644
--- a/drivers/tee/tee_core.c
+++ b/drivers/tee/tee_core.c
@@ -1189,6 +1189,7 @@ int tee_client_cancel_req(struct tee_context *ctx,
 	return ctx->teedev->desc->ops->cancel_req(ctx, arg->cancel_id,
 						  arg->session);
 }
+EXPORT_SYMBOL_GPL(tee_client_cancel_req);
 
 static int tee_client_device_match(struct device *dev,
 				   struct device_driver *drv)
-- 
2.25.1

